cmd/ctr/commands/resolver.go

```diff
@@ -117,8 +118,7 @@ func GetResolver(ctx gocontext.Context, clicontext *cli.Context) (remotes.Resolv
 }
 
 func resolverDefaultTLS(clicontext *cli.Context) (*tls.Config, error) {
-	config := &tls.Config{}
-
+	config := internal.TLSConfig()
```

Comment:
Since this `*tls.Config` is used in a client context, the `MinVersion: tls.VersionTLS10` in `internal.TLSConfig()` regresses behavior somewhat in that we will additionally support fetching/pushing from registries that do not support a modern TLS version.